lib/protocol/networks.js

```diff
@@ -899,23 +897,23 @@ simnet.pow.targetTimespan =
   simnet.pow.targetWindow * simnet.pow.targetSpacing;
 simnet.pow.minActual = (simnet.pow.targetTimespan / 4) >>> 0;
 simnet.pow.maxActual = simnet.pow.targetTimespan * 4;
-simnet.pow.targetReset = false;
+simnet.pow.targetReset = true;
 simnet.pow.noRetargeting = false;
```

Comment:
I opted into `targetReset` to allow the pow difficulty to reset if there were no blocks produced after a very long period. This is useful since there is a GPU miner out there and there was not always consistent mining on `testnet4-stable`.